README.md

```diff
@@ -1,3 +1,29 @@
-# Paper Extension
-A chrome extension to sync the highlighted text from web to a dropbox paper document. 
+<h1  align='center'>
+Paper extension
+</h1> 
+<p align='center'>
+<img src='./img/icon-large.png' >
+</p>
+<h5 align='center'>
+Take notes without leaving the tab 
```

Comment:
I like this caption! But we should also include the current subtitle somewhere as it explains technically what our project does.